desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/PathBrowser.tsx

```diff
@@ -54,6 +56,9 @@ const PathBrowser = ({
     s3: <S3Icon />
   };
 
+  const { fileSystem, path } = useMemo(() => getFileSystemAndPath(filePath), [filePath]);
```

Comment:
You probably don't need useMemo for getFileSystemAndPath since the computation is so lightweight and the result us destructed into primitive types before accessed.

Reply:
Make sense. Fixed